assets/js/atomic/blocks/product-elements/product-image-gallery/editor.scss

```diff
@@ -0,0 +1,12 @@
+.wc-block-editor-product-gallery {
+	img {
+		width: 500px;
+		height: 500px;
+	}
+	.wc-block-editor-product-gallery__gallery {
```

Comment:
I am nitpicking here but maybe we can improve the naming of this class. Some of alternatives could be:
- `wc-block-editor-product-gallery__alternative-images`
- `wc-block-editor-product-gallery__other-images`

If it doesn't break the frontend.

Reply:
Thanks for the suggestion! I picked `wc-block-editor-product-gallery__other-images`. [795a698](https://github.com/woocommerce/woocommerce-blocks/pull/8235/commits/795a6980d6e084b183b7499f163689df18e4bad2)